content/pages/healthcare/apply/application.md

```diff
@@ -3,9 +3,10 @@ title: Apply for Health Care
 entryname: hca
 layout: page-react.html
 body_class: page-healthcare
-in_maintenance: false
-maintenance_line1: We're sorry. The healthcare application is currently down while we fix a few things. We will be back up as soon as we can.
-maintenance_line2: In the meantime, you can still call 1-877-222-VETS(8387) and press 2 to complete this application over the phone.
+in_maintenance: true
+maintenance_line1: Scheduled Maintenance
+maintenance_line2: Please try again after 12pm EST Saturday, January 7th.
```

Comment:
Also, really sorry this is my fault, but I got the dates wrong. It's Sunday, January 8th they're doing maintenance, not Saturday. Sorry!